x/mint/types/minter.go

```diff
@@ -8,18 +8,18 @@ import (
 	"github.com/Nolus-Protocol/nolus-core/custom/util"
 )
 
-// Minting formula f(x)=-4.33275 x^3 + 944.61206 x^2 - 88567.25194 x + 3.86335×10^6 integrated over 0.47 to 96
-// afterwards minting 103125 tokens each month until reaching the minting cap of 150*10^6 tokens.
+// Legacy Minting formula integral-4.33275 x^3 + 952.82456 x^2 - 88567.49981 x + 3.86381×10^6 dx = -1.08319 x^4 + 317.608 x^3 - 44283.7 x^2 + 3863810 x + constant.
+
+// Current Minting formula integral-0.11175 x^3 + 50.82456 x^2 - 1767.49981 x + 0.83381×10^6 dx = -0.0279375 x^4 + 16.9415 x^3 - 883.75 x^2 + 833810 x + constant.
 var (
-	QuadCoef          = sdkmath.LegacyMustNewDecFromStr("-1.08319")
-	CubeCoef          = sdkmath.LegacyMustNewDecFromStr("314.871")
-	SquareCoef        = sdkmath.LegacyMustNewDecFromStr("-44283.6")
-	Coef              = sdkmath.LegacyMustNewDecFromStr("3863350")
-	MintingCap        = util.ConvertToMicroNolusInt64(150000000)
+	QuadCoef          = sdkmath.LegacyMustNewDecFromStr("-0.0279375")
+	CubeCoef          = sdkmath.LegacyMustNewDecFromStr("16.9415")
+	SquareCoef        = sdkmath.LegacyMustNewDecFromStr("-883.75")
+	Coef              = sdkmath.LegacyMustNewDecFromStr("833810")
 	FixedMintedAmount = util.ConvertToMicroNolusInt64(103125)
```

Comment:
I believe this is unused ... do we run some form of static code analysis tool?

Reply:
golangci linter should be catching those. I will add some linters for this like:
https://golangci-lint.run/usage/linters/#ineffassign
https://golangci-lint.run/usage/linters/#unused